include/MVC/Controller/SugarController.php

```diff
@@ -411,6 +412,48 @@ protected function handleException(Exception $e)
         }
     }
 
+    /**
+     * Handle SuiteCRM Exceptions
+     * @param \SuiteCRM\Exception\Exception $exception
+     */
+    protected function handleSuiteCRMException(\SuiteCRM\Exception\Exception $exception)
```

Comment:
Typehinting is only supported from PHP 7.0 onwards, but SuiteCRM 7.10 should be compatible with 5.5.9 and onwards according to the compatibility matrix.

Reply:
Please refer to [php documention](http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration.types). As of php 5.0 you can use classes/interfaces. however primative types like string, int, float are on only for php 7.0 and above. Keep in mind that I have tested this code on PHP 5.5.9 - which is the minimum requirements for SuiteCRM 7.10

Reply:
My bad, you are absolutely right